redbeams/src/main/java/com/sequenceiq/redbeams/exception/BadRequestException.java

```diff
@@ -0,0 +1,30 @@
+package com.sequenceiq.redbeams.exception;
+
+import java.util.Objects;
+
+public class BadRequestException extends RuntimeException {
+
+    public BadRequestException(String message) {
+        super(message);
+    }
+
+    public BadRequestException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
I wonder if this class needs equals and hashCode methods. Now is a good time to check if you are inclined to.